source/_components/switch.xiaomi_vacuum

```diff
@@ -0,0 +1,47 @@
+The `xiaomi_vacuum`switch platforms allows you to control the state of your [Xiaomi Mi Robot Vacuum](http://www.mi.com/roomrobot/). 
+Current supported features are start and stop (go to dock).
+
+{% linkable_title Installation %}
```

Comment:
The installation should be unnecessary as homeassistant downloads the dependencies.